frontend/src/components/AccountReceiveTable.vue

```diff
@@ -233,15 +233,23 @@
               <!-- Sender column -->
               <div v-else-if="col.name === 'from'" class="d-inline-block">
                 <div @click="copyAddress(props.row.from, 'Sender')" class="cursor-pointer copy-icon-parent">
-                  <span>{{ col.value }}</span>
+                  <span
+                    ><a :href="getSenderOrReceiverEtherscanUrl(col.value)" target="_blank">{{
```

Comment:
Looks like some formatting got messed up? Right now it has the default browser style which looks ugly IMO

<img width="311" alt="image" src="https://user-images.githubusercontent.com/17163988/186034103-e9c7fd86-05d6-4851-9e56-7b3c08e4908b.png">

And it also cuts off the info, whereas before it looked like this

<img width="357" alt="image" src="https://user-images.githubusercontent.com/17163988/186034782-b811cf1b-57a9-4930-9f0f-f6a6af640684.png">

You may want to add `class="hyperlink"` which is the style we usually give to hyperlinks. However, I think that would get noisy with a big table so it may be better to just keep the regular text style

Reply:
I actually do prefer "hyperlink" styling

Reply:
You mean the hyperlink class, not the default blue style? If so that's fine, though we do still need to fix the "sender" name/addr being truncated like in the first screenshot

Reply:
Yea, I like the golden link style. Truncation of ens names should be good now.